src/cromshell/utilities/io_utils.py

```diff
@@ -101,18 +101,28 @@ def is_workflow_id_valid(workflow_id: str):
     return workflow_id_pattern.match(workflow_id)
 
 
-def pretty_print_json(json_text: str):
+def color_json(formatted_json: str) -> str:
+    return highlight(formatted_json, lexers.JsonLexer(), formatters.TerminalFormatter())
+
+
+def pretty_print_json(format_json: str or dict, add_color: bool = False):
     """Prints JSON String in a fancy way
 
-    - json_text: content of the json, NOT json file path"""
+    - json_text: valid json string or dictionary, NOT json file path"""
 
-    loaded_json = json.loads(json_text)
+    if format_json is type(str):
+        loaded_json = json.loads(format_json)
+    else:
+        loaded_json = format_json
     pretty_json = json.dumps(loaded_json, indent=4, sort_keys=True)
-    print(pretty_json)
+    if add_color:
+        print(color_json(pretty_json))
```

Comment:
should logger be used here, or print?

Reply:
"Logging is a means of tracking events that happen when some software runs. The software’s developer adds logging calls to their code to indicate that certain events have occurred. " ([link](https://docs.python.org/3/howto/logging.html))

The JSON being printed isn't an event but an output of the command, I don't think Logger would be right given this description